kbase-extension/static/kbase/js/widgets/appWidgets2/input/sequenceInput.js

```diff
@@ -419,35 +379,25 @@ define([
                 container = parent.appendChild(document.createElement('div'));
                 ui = UI.make({ node: container });
 
-                return render(config.initialValue).then(() => {
-                    channel.on('reset-to-defaults', () => {
-                        resetModelValue();
-                    });
-                    channel.on('update', (message) => {
-                        setModelValue(message.value);
-                    });
-                    channel.on('refresh', () => {});
```

Comment:
again, model is unused, other than setting the values. Might need to have some work done to reroute the `update` calls, but I'm not sure they're necessary? This has been in production for some years now...